newt/toolchain/deps.go

```diff
@@ -244,6 +272,7 @@ func (tracker *DepTracker) CompileRequired(srcFile string,
 		}
 
 		if depModTime.After(objModTime) {
+			logRebuildReqdNewDep(srcFile, dep)
 			util.StatusMessage(util.VERBOSITY_VERBOSE, "%s - rebuild required; obj older than dependency (%s)\n", srcFile, dep)
```

Comment:
This line can be removed, no?

Reply:
Right you are.  Thanks, @utzig.  I have removed the extra log line,